### PR TITLE
Capstone Project - Youth Summits Website

### PR DESCRIPTION
**Pull request by yemidada**
In this project, I
- create a summit website with bootstrap template framework
- Implement a popup menu on a mobile view
- Connected my designs to github page
- Use CSS grid and flex for my designs
- Use semantic HTML tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,193 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can support us  by sharing our code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks you microverse for the opinion to opportunity
```
Comment (nedjwab) [inline README.md]: [ ] Kindly update your readme file, Add gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <div class="popup-menu">
+            <img class="popup-close" id="popup-close-btn" src="./assets/images/popup-close-white.svg" />
+            <ul>
+                <li class="popup-menu-item">
+                    <a class="nav-link" href="about.html">About</a>
```
Comment (nedjwab) [inline about.html:71]: [ ] kindly Fix your navbar, so It's possible to navigate between the pages, there's a link from the Home page to the About page and the opposite too.🤓
your design: 👇
about

Figma design: 👇
design

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @yemidada  

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- I like Your Project , the Design and The code are amazing ✨🔥🔥🔥🔥

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 
 👇👇👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by osamaashraf6 — APPROVED** [review]
Hi @yemidada 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (2)

## Highlights🎉🎉
- [x] No linter errors present👌
- [x] The PR has a proper title and a descriptive body 👍
- [x] Nice design 💯 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**